aiidalab_eln/base_connector.py

```diff
@@ -14,8 +14,6 @@ def __init__(self, **kwargs):
 
         Args:
             eln_instance (str): URL which points to the eln instance.
-            user (str): ELN user.
```

Comment:
maybe username would have been a more explicit name, but not too important, maybe to mention in the docstring